content/asset-tracking/using-the-sdks.textile

```diff
@@ -0,0 +1,232 @@
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Supported platforms#supported-platforms
+    - Repositories#repositories
+    - Prerequisites#prerequisites
+    - Installing the SDK#installing-sdk
+    - Using the Publisher SDK#publisher-sdk
+    - Using the Subscriber SDK#subscriber-sdk
+    - See also#see-also
+---
+
+This topic explains how to get started using the Ably Asset Tracking SDKs.
+
+h2(#supported-platforms). Supported platforms
+
+There are two Asset Tracking SDKs, one for publishing and one for subscribing. The following platforms are supported:
+
+* Android (Java and Kotlin) - publisher and subscriber SDK
+* iOS (Objective-C and Swift) - publisher and subscriber SDK
+* Web (JavaScript) - subscriber SDK
+
+h2(#repositories). SDK repositories
+
+The SDKs can be found in the following GitHub repositories:
+
+* "Android":https://github.com/ably/ably-asset-tracking-android
+* "iOS":https://github.com/ably/ably-asset-tracking-cocoa
+* "JavaScript":https://github.com/ably/ably-asset-tracking-js
+
+h2(#prerequisites). Prerequisites
+
+You need to have a suitable developer environment installed, for example:
+
+* Android - "Android Studio":https://developer.android.com/studio
+* iOS - "Xcode":https://developer.apple.com/xcode/
+* JavaScript - any suitable environment of your choice 
+
+You also need to have suitable credentials for the various SDK components:
+
+* @ABLY_API_KEY@ - Your Ably API key
+* @MAPBOX_DOWNLOADS_TOKEN@ - Mapbox credentials
+* @MAPBOX_ACCESS_TOKEN@ - Mapbox credentials
+* @GOOGLE_MAPS_API_KEY@ - Google Maps credentials
+
+You can set these values in your @~/.gradle/gradle.properties@ file.
+
+h2(#installing-sdk). Installing the SDK
+
+You can find information on installing the SDKs in the following resources:
+
+* "Instructions for Android":https://github.com/ably/ably-asset-tracking-android
+* "Instructions for iOS":https://github.com/ably/ably-asset-tracking-cocoa
+* "Instructions for JavaScript":https://github.com/ably/ably-asset-tracking-js
+
+h2(#publisher-sdk). Using the publisher SDK
+
+Common operations you will need to carry out on the publisher include:
+
+* Initialization of the publisher.
+* Start tracking an asset.
+* Stop tracking an asset.
+* Set the resolution constraints on an asset.
+
+h3(#publisher-initializing). Initializing the Publisher
+
+During initialization of the publisher various methods can be called to configure the @Publisher@.
+
+The required methods are:
+
+- connection := Called to provide Ably connection information, such as API keys, and any other configuration parameters as needed. 
+- map := Called to provide Mapbox configuration, such as API keys, any other configuration parameters as needed. 
+- resolutionPolicy := Sets the policy factory to be used to define the target resolution for publishers created from this builder.
+
+The optional methods are:
```

Comment:
One optional method is missing - `locationSource`

Reply:
Fixed. `locationSource` added to optional methods.